cgi/auth.pl

```diff
@@ -63,5 +85,12 @@
 	$r->err_headers_out->set("Access-Control-Allow-Origin", $origin);
 }
 
+print header(-status => $status, -type => 'application/json', -charset => 'utf-8');
+print $json;
+
 $r->rflush;
-$r->status($status);
+
+# Setting the status makes mod_perl append a default error to the body
+# $r->status($status);
+# Send 200 instead.
+$r->status(200);
```

Comment:
Hum ain't it too big a change for third party app ? (javascript app etc.).

Maybe we should return json only if the user add a json=1 parameter ? (and in this case return a 200).

Reply:
The only apps that use /cgi/auth.pl are Folksonomy Engine and Hunger Games I believe (it's not documented anywhere). And I think they only look at the status code, as we were only returning a status code, with an empty body.

Reply:
Note that this $r->status(200) actually does not change the true return status that is sent to the client, which is set in the header above.

![image](https://user-images.githubusercontent.com/8158668/194600691-6a7e5c78-1ee9-476d-8f85-09f68fb230ff.png)